README.md

```diff
@@ -16,8 +16,23 @@ yarn upgrade webpack-dev-server --latest
 yarn install
 bin/rails test
 ```
+2. Enable `unsafe-eval rule` for development environment
```

Comment:
Please use in title why do we need this instead of what. 

This is installation of webpacke+vue.js step